woocommerce-services.php

```diff
@@ -258,6 +258,9 @@ public static function plugin_uninstall() {
 
 		/**
 		 * Deletes WC Admin notices.
+		 *
+		 * @todo We'd need to make sure the DHL Live Rates class reachable during uninstall.
+		 *       We should be able to require everything in the 'pre_uninstall_plugin' action, so it shouldn't be an issue.
```

Comment:
I'm not sure how to test this. Can you give me some testing steps @kallehauge , please? Thanks!

Reply:
No need @Ferdev ! My concern was: if someone uninstalled WCS&T, then if we had not initiated shipping logic, then this shouldn't cause a fatal error because the class doesn't exist.

TL;DR: It's not an issue since we require the class right before we call it, and the method is static and doesn't depend on anything outside core Woo code, so we're good here! 👏